packages/ui/src/components/organisms/ChipSelect/ChipSelect.tsx

```diff
@@ -65,6 +68,25 @@ const ChipSelect = ({
     }
   }, [isRefresh, selectionItems]);
 
+  const _width = useMemo(() => {
+    if (width != null) {
+      return width;
+    }
+
+    const numberItems = selectionItems.length;
+
+    if (numberItems > 1 && numberItems < 5) {
+      return (
+        (Dimensions.get('window').width -
+          CHIP_CONTAINER_MARGIN * 2 -
+          CHIP_MARGIN * 2 * numberItems) *
+        (1 / numberItems)
+      );
+    }
```

Comment:
If only one chip, component should return null, if more than 5, chip should adjust to content length